mobile/ios/ios-proxy-services.ts

```diff
@@ -392,13 +391,17 @@ export class IOSSyslog {
 	}
 
 	public read(): void {
+		var shouldLog = false;
+		setTimeout(() => shouldLog = true, 2500);
+
 		var printData = (data: NodeBuffer) => {
-			var output = ref.readCString(data, 0);
-			if(this.matchRegex.test(output)) {
-				this.$logger.out(output);
+			if(shouldLog) {
+				var output = ref.readCString(data, 0);
+				process.stdout.write(output);
```

Comment:
Because if we use `logger.out` we produce two empty lines on console. Actually `output` ends with `\n` and `logger.out` adds one more `\n`. `process.stdout.write` doesn't add additional empty line. :smile: